spec/features/add_peep_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'pg'
+
+feature 'add new peep' do
+  scenario 'user can add new peeps to the page' do
+    connection = PG.connect(dbname: 'chitter_data_test')
+
```

Comment:
Good use of testing database, allows you to make almost a 'double' to test your page.